### ISSUE TITLE
Import "helper_functions" could not be resolved(reportMissingImports)

### ISSUE THREAD

**Issue by rafaelblevin821** [mrdbourke/pytorch-deep-learning#329]
Hi, I am getting this error message on 03. PyTorch computer vision 6.1. 

`Import "helper_functions" could not be resolved(reportMissingImports)`

Any suggestions on how to resolve?

**Comment by PrzemyslawCh** [comment]
Have you solved your problem? I have same error and after 2 days everything start working. It was a problem with colab itself i guess.

**Comment by rafaelblevin821** [comment]
> Have you solved your problem? I have same error and after 2 days everything start working. It was a problem with colab itself i guess.

No, it's still not working.

**Comment by PrzemyslawCh** [comment]
It is not a problem, actually. Google Colab writes this, but you can access functions from helper_functions anyway. I have many similar small problems, but code still works So. Just I just move on.

**Comment by rafaelblevin821** [comment]
> It is not a problem, actually. Google Colab writes this, but you can access functions from helper_functions anyway. I have many similar small problems, but code still works So. Just I just move on.

Yes, it still works, but the weights are not updating correctly.

**Comment by mrdbourke** [comment]
Hi @rafaelblevin821 and @PrzemyslawCh,

I'm not sure what's happening here.

If it's just a warning but the code works, feel free to ignore it.

Otherwise I can confirm that downloading/importing the `helper_functions.py` file works for me.

See more here:
• Answer on downloading `helper_functions.py` - https://github.com/mrdbourke/pytorch-deep-learning/discussions/308#discussioncomment-5096916 
• Example notebook for downloading `helper_functions.py` - https://colab.research.google.com/drive/1WgEpv0VhI9Wo_dizMMwNuX1AzLtTrDeJ?usp=sharing
